packages/core/src/Table/TableCellHead/Sorter.js

```diff
@@ -0,0 +1,64 @@
+import { requiredIf } from '@dhis2/prop-types'
+import { colors } from '@dhis2/ui-constants'
+import PropTypes from 'prop-types'
+import React from 'react'
+import { Action } from './Action'
+
+export const ASC = 'asc'
+export const DESC = 'desc'
+export const DEFAULT = 'default'
+export const SORT_DIRECTIONS = [DEFAULT, ASC, DESC]
+
+export const getNextSortDirection = currentDirection => {
+    const currentIndex = SORT_DIRECTIONS.indexOf(currentDirection)
+    const endIndex = SORT_DIRECTIONS.length - 1
+    const nextIndex = currentIndex === endIndex ? 0 : currentIndex + 1
+
+    return SORT_DIRECTIONS[nextIndex]
+}
+
+export const Sorter = ({ name, sortDirection, onSortIconClick }) => {
+    const nextSortDirection = getNextSortDirection(sortDirection)
+    const handleSortIconClick = event => {
+        onSortIconClick({ name, direction: nextSortDirection }, event)
+    }
+
+    return (
+        <Action onClick={handleSortIconClick}>
```

Comment:
Maybe we should have a `title` attribute with value `Sort values` or some text that is hidden visually but visible to screen readers?